pkg/dispatchcli/cmd/delete.go

```diff
@@ -38,10 +38,10 @@ func NewCmdDelete(out io.Writer, errOut io.Writer) *cobra.Command {
 			}
 
 			deleteMap := map[string]modelAction{
-				"image":      CallDeleteImage,
-				"base-image": CallDeleteBaseImage,
-				"function":   CallDeleteFunction,
-				"secret":     CallDeleteSecret,
+				"Image":     CallDeleteImage,
```

Comment:
utils.ImageKind and similar for keys below

Reply:
Yeah, I just noticed this in working on my CallUpdateImage changes. I have noted this to be fixed later. thanks.

Reply:
Np!